.github/CODEOWNERS

```diff
@@ -4,3 +4,10 @@
 # review when someone opens a pull request.
 
 *       @department-of-veterans-affairs/backend-review-group
+
+# Sample CSVs
+sample_csvs @department-of-veterans-affairs/afs-code-reviewers
+
+# Github config
+.github/PULL_REQUEST_TEMPLATE.md @department-of-veterans-affairs/afs-code-reviewers
+
```

Comment:
I'm checking with the team on code ownership rules. In the meantime, I believe that you want to add @department-of-veterans-affairs/backend-review-group to both of these entries, which would make the code owner approval an OR statement. Right now, this line as is would give @department-of-veterans-affairs/afs-code-reviewers sole ownership of the two files listed. In the case that we have an emergency situation, we would want the ability for @department-of-veterans-affairs/backend-review-group to approve/merge code.

Reply:
Thanks @LindseySaari  . I have added backend-review-group to the CODEOWNERS file.

I'm only suggesting AFS because BAH, which was the vendor before we took over, also had it. Here's the PR where they remove themselves before rolling off:

https://github.com/department-of-veterans-affairs/gibct-data-service/pull/861

Reply:
Thanks for the link! Going to approve!

Reply:
Ty!

Reply:
@LindseySaari  It looks like there's a vulnerability issue so Jenkins isn't building out the project. I also put a fix for that so Jenkins can build it out. Can you please approve again?

Reply:
@LindseySaari  can you merge this in?